### PR TITLE
Rebase form fields orientation

### PR DESCRIPTION
**Pull request by sampila**

### CODE REVIEW

File: forms/pdf_form_fields_rotations.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
+/*
+ * Add images to a PDF file, one image per page.
```
Comment (gunnsth) [inline forms/pdf_form_fields_rotations.go:2]: @sampila maybe update the description and indicate how this is related to forms?

**Review by gunnsth — CHANGES_REQUESTED** [review]
added a comment

File: forms/pdf_form_fields_rotations.go (hunk 1 above)
Comment (sampila) [reply forms/pdf_form_fields_rotations.go]: Updated at 103bfc2cb8d0ec2ff69315f3023f9df9f3132268

File: forms/pdf_form_fields_rotations.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
+/*
+ * Get form fields appearance rotation value.
```
Comment (gunnsth) [inline forms/pdf_form_fields_rotations.go:2]: hm, what is the relation to images then?

Comment (sampila) [reply forms/pdf_form_fields_rotations.go]: Updated at e3c009fe45cbb4df9bd4d909958454d04997382d

**Review by gunnsth — APPROVED** [review]

**Pull request merged**